ansible/roles/java-base/tasks/main.yml

```diff
@@ -61,6 +61,27 @@
     dest: "/usr/bin/java"
     state: link
 
+ - name: download java on s390x
+   when: os == "rhel72" and arch == "s390x"
```

Comment:
Aren’t the new machines `rhel7` (without the `2`)?

Reply:
Yes they are

Reply:
Do you have a conditional that's used elsewhere for "Linux" generally? That's what I was aiming for (Or basically "s390x & !zos")